modules/nw-anp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-audit-logging_{context}"]
+= AdminNetworkPolicy audit logging
+
+Audit logging is enabled per `AdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
+
+.Example of annotation for `AdminNetworkPolicy` CR
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  annotations:
+    k8s.ovn.org/acl-logging: '{ "deny": "alert", "allow": "alert", "pass" : "warning" }'
+  name: anp-tenant-log
+spec:
+  priority: 5
+  subject:
+    namespaces:
+      matchLabels:
+        kubernetes.io/metadata.name: tenant
+  egress:
+    - name: deny-ns-tenant-1
+      action: Deny
+      to:
+      - namespaces:
+           matchLabels:
+             kubernetes.io/metadata.name: tenant-1
+    - name: allow-ns-tenant-2
+      action: Allow
+      to:
+      - namespaces:
+           matchLabels:
+             kubernetes.io/metadata.name: tenant-2
+    - name: pass-ns-tenant-3
+      action: Pass
+      to:
+      - namespaces:
+           matchLabels:
+             kubernetes.io/metadata.name: tenant-3
+----
+====
+
+Logs are generated whenever a specific ACL meets the criteria set in your annotation. In the example, an event in which the `tenant` namespace is denied egress to the `tenant_1` namespace a log is generated.
+
+The logging format is compatible with syslog as defined by RFC5424. The syslog facility is configurable and defaults to `local0`. An example log entry might resemble the following:
```

Comment:
Yes, it is. I wasn't sure if it should be included or not. I thought not duplicating meant that we were assuming the user understood something. I thought it was perhaps better to make it explicit rather than left implicit.

Reply:
I think since its under the umbrella main section of "audit logging" it is already explicit and apparent so IMHO no need to call this out in every section.